components/quickbooks/sources/custom-webhook-events/custom-webhook-events.js

```diff
@@ -0,0 +1,48 @@
+const quickbooks = require("../../quickbooks.app");
+const common = require("../common");
+
+module.exports = {
+  ...common,
+  key: "quickbooks-custom-webhook-events",
+  name: "Custom Set of Webhook Entities (Created, Updated, Merged, Deleted, Voided or Emailed)",
```

Comment:
The [current pattern](https://pipedream.com/docs/components/guidelines/#source-name) to identify a real-time event source is to include “(Instant)” in the source name. E.g., “New Search Mention” or “New Submission (Instant)”.

```suggestion
  name: "Custom Set of Webhook Entities (Created, Updated, Merged, Deleted, Voided or Emailed) (Instant)",
```